src/screens/Home/Home.tsx

```diff
@@ -33,6 +33,9 @@ const Home: React.FC<THome> = () => {
           />
         </View>
       </View>
+      <View style={{ marginTop: 20 }}>
```

Comment:
Don't use inline styles, and  `marginTop: 20` should be using from spacing for responsiveness.